packages/rxsocketio/src/operators/conduit.js

```diff
@@ -7,6 +7,7 @@ import {
   pairwise,
```

Comment:
⏬ This code is mostly the same but I broke it up a bit so I could test the pieces more easily.

Reply:
That follows general best practices for a functional based development approach. Which is concise functions that have a single job that over large complex functions with with multiple jobs.